README.md

```diff
@@ -15,6 +15,11 @@ This filter provides Graphviz code block support.
 - graphviz
 - dot2tex (pdf/LaTeX support)
 
+On Debian, you can install the required packages as follows:
+    ```
+    $ sudo apt install ghc libghc-utf8-string-dev libghc-sha-dev libghc-pandoc-dev
```

Comment:
Shouldn't `graphviz` and `dot2tex` be included here?

Reply:
They are pulled in as dependencies, I only added the minimum amount of packages required. The dependency management handles the rest.

Reply:
I meant we should also include `pandoc`, `graphviz`, and `dot2tex` in the instruction. I tried your command but it didn't pull these three packages.

Reply:
OK, I've added these packages to the overall list. Thanks!